components/prism-php.js

```diff
@@ -33,6 +33,14 @@
 				'punctuation': /\\/
 			}
 		},
+		'class-name-definition': [
+			{
+				pattern: /(\b(?:class|interface|trait)\s+)\b[a-z_]\w*(?!\\)\b/i,
+				greedy: true,
```

Comment:
I don't think greedy is necessary, is it?

Reply:
Yes, I think that greedy can be dropped.